ddp-workspace/projects/ddp-mpc/src/app/components/welcome/welcome.component.ts

```diff
@@ -0,0 +1,242 @@
+import { Component, OnInit } from '@angular/core';
+import { GoogleAnalyticsEventsService, BrowserContentService, WindowRef, GoogleAnalytics } from 'ddp-sdk';
+
+@Component({
+    selector: 'welcome',
+    template: `
+    <toolkit-header [showButtons]="true"></toolkit-header>
+    <div class="Wrapper">
```

Comment:
Yeah... I also have noticed that issue on the small screens. Since we use a common header component for all apps, this problem also occurs on angio, brain, mbc, esc(gec).
My idea - we can hide Count me in button on the small screens and show it on the side menu:
![image](https://user-images.githubusercontent.com/30126907/75767853-83514980-5d54-11ea-892d-554428f83c3f.png)
What do you think?

Reply:
IMO, the "count me in" button is considered and important piece of CMI's branding, so I think storing the functionality inside a menu would not be well received by users.
Just looking at the IPhone screenshot above, it looks like we ought to have enough horizontal pixels to slide it in.  If need be, the MPC logo could slide a bit to the left too. Make sense?

Reply:
Hm, I think even with these actions menu button will be hidden... Bit I'll try to do some magic with positions, margins and paddings.

Reply:
@MocanaAtBroad, I've played with margins and paddings even with zero margins and paddings, buttons are clipped on iPhone 5/SE and other small screen devices. Yesterday I discussed it with Jen and Erin. they agree to move buttons under the side menu(in addition, this approach is already used in osteo), so, I'll clarify the requirements and transform them into the separate ticket.

Reply:
I should have suggested to speak to Erin in first place!

Reply:
Fix for this is still pending, right?

Reply:
Yes, but not in this ticket's scope.